tests/quantization/utils.py

```diff
@@ -4,10 +4,7 @@
 
 def is_quant_method_supported(quant_method: str) -> bool:
     # Currently, all quantization methods require Nvidia or AMD GPUs
-    if not (current_platform.is_cuda() or current_platform.is_rocm()):
-        return False
+    capability = int(current_platform.get_device_capability() or -1)
```

Comment:
this seems to change the semantic?

Reply:
Currently, `current_platform.get_device_capability()` return `None` except for cuda and rocm backends. So, the result is the same.

Reply:
I assume that quantization support is based on the device capability.

Reply:
I think previous code is more intuitive. you can just replace the capability comparison with the new `has_device_capability` , while keeping `if not (current_platform.is_cuda() or current_platform.is_rocm()):`